srcpkgs/mopidy-spotify/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'mopidy-spotify'
+pkgname=mopidy-spotify
+version=4.0.1
+revision=1
+wrksrc="Mopidy-Spotify-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-setuptools"
+depends="mopidy libspotify python3-pyspotify"
```

Comment:
libspotify does not need to be listed here, it will be installed as a transitive dependency via python3-pyspotify.

Reply:
Totally true. Fixed